tools/deviceGen/output/features_members.h

```diff
@@ -0,0 +1,1581 @@
+    /*
+       Vulkan 1.0 Core
+    */
+    bool robustBufferAccess = false; // widely supported but has performance overhead, so remains an optional feature to enable
+    /*
+       [REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance
+    */
+    // bool fullDrawIndexUint32 = true;
+    // bool imageCubeArray = true;
+    // bool independentBlend = true;
+    /*
+       I have no clue if these cause overheads from simply being enabled
+    */
+    bool geometryShader = false;
+    bool tessellationShader = false;
+    // bool sampleRateShading = true; // [REQUIRE] ROADMAP 2022 and good device support
+    // bool dualSrcBlend = true; // [REQUIRE] good device support
+    // bool logicOp; // [EXPOSE AS A LIMIT] somewhat legacy features
+    /*
+       [REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance
+    */
+    // bool multiDrawIndirect = true;
+    // bool drawIndirectFirstInstance = true;
+    // bool depthClamp = true;
+    // bool depthBiasClamp = true;
+    // bool fillModeNonSolid = true; // [REQUIRE] good device support
+    bool depthBounds = false; // this is kinda like a weird clip-plane that doesn't count towards clip plane count
```

Comment:
proper grouping in the JSON, and a newline between groups in the output would really help with readability here

Reply:
also don't emit the comment AFTER the line, emit it before